### ISSUE TITLE
subnet_ids

### ISSUE THREAD

**Issue by sermolin** [aws-samples/amazon-sagemaker-w-snowflake-as-datasource#8]
I like your neat trick of autodiscovering subnet_ids.
However, not all subnets in a vpc could be suitable for deploying training instances (think of Denali). Also not all subnets could be connected to each other. In case of DDP, is it possible for instances may end up being deployed in a different subnets and not able to communicate to each other.

**Comment by aarora79** [comment]
I don't think it is possible that individual instances get deployed in different subnets and then cannot communicate with each other. @divyamurali168 please confirm.
